__tests__/controllers/users.controller.spec.ts

```diff
@@ -0,0 +1,65 @@
+import * as Faker from 'faker';
+import request from 'supertest';
+import connection from '../../src/database/connection';
+import { getRepository } from 'typeorm';
+import { User } from '../../src/entities/user.entity';
+import app from '../../src/app';
```

Comment:
```suggestion
import * as Faker from 'faker';
import request from 'supertest';
import { getRepository } from 'typeorm';

import connection from '../../src/database/connection';
import { User } from '../../src/entities/user.entity';
import app from '../../src/app';
```
Maybe you can re-order the imports in this way  _you can dismiss this comment_

Reply:
I think ts-paths are not working inside `__test__` folder @alealvarez14 ?